### DISCUSSION TITLE
Alist 数字证书 应该选哪种格式？

### DISCUSSION THREAD

**Discussion by at2008** [AlistGo/alist#298] [Q&A] [answered]
用阿里云提供的免费证书，有多种格式：
apache、nginx、iis、tomcat等等，

config.json 文件也改过了：

  "https": true,
  "cert_file": "/root/ca/nginx/xx.pem",
  "key_file": "/root/ca/nginx/xx.key",

不生效，还是只能http；

是不是必须443端口？

**Comment by xhofe** [comment, marked as answer]
无需必须443端口，`cert_file`和`key_file`文件后缀分别应为`.crt`和`.key`。

**Reply by xhofe** [reply]
nginx中的。

**Reply by at2008** [reply]
阿里提供的nginx 的证书 是 xx.pem 和 xx.key ;
配置里面输入的就是nginx的；也重启服务了，默认还是http，https无法访问

**Reply by xhofe** [reply]
测试了使用pem和key文件https正常，并无问题。

**Reply by at2008** [reply]
好的，多谢回复，我自己再试试。

**Reply by at2008** [reply]
更新配置文件后问题解决：

  "scheme":{
	"https": true,
	"cert_file": "/root/ca/nginx/hao.pem",
	"key_file": "/root/ca/nginx/hao.key"
  },
  "cache":{
	"expiration": 60,
	"cleanup_interval": 120
  }
